docs/CSS.md

```diff
@@ -0,0 +1,138 @@
+### 1. `float`
+
+* *elements:* all
+* *inherited:* no
+
+#### Values
+
+* `none` (default) -- Is a keyword indicating that the element must not float.
+* `left` -- Is a keyword indicating that the element must float on the left side of its containing block.
+* `right` -- Is a keyword indicating that the element must float on the right side of its containing block.
+
+### 2. `margin`
+
+* *elements:* all
+* *inherited:* no
+
+#### Values
+
+* `length` -- Specifies a fixed width. Negative Values are allowed. See for <length> possible units.
+* `percentage` -- A percentage relative to the width of the containing block. Negative values are allowed.
+* `auto` --  is replaced by some suitable value, e.g. it can be used for centering of blocks.
+div { width:50%;  margin:0 auto; } centers the div container horizontally.
+
+### 3. `max-width`
+
+* *elements:* all
+* *inherited:* no
+
+#### Values
+
+
+* `none` (default) -- The width has no maximum value.
+* `length` -- See length for possible units.
+* `percentage`-- Specified as a percentage of containing block's width.
+* `max-content` -- The intrinsic preferred width.
+* `min-content` --The intrinsic minimum width.
+* `fill-available` --The containing block width minus horizontal margin, border and padding. Some browsers implement an ancient name for this keyword, available.
+* `fit-content` --The same as max-content.
```

Comment:
It would be good to include compatibility information (the "supported" field from the HTML assignment), since CSS properties and their values are tricky.